### PR TITLE
[MIG] website_disable_odoo

### PR DESCRIPTION
**Pull request by cubells**
New module to remove odoo.com bindings on website

### CODE REVIEW

**Comment by cubells** [comment]
cc @Tecnativa

**Comment by rafaelbn** [comment]
Thanks! please check travis error

**Comment by yajo** [comment]
@cubells I'm pretty sure you should have created a separate git branch instead of making the PR directly from 9.0; just take note for future PR :wink:

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
-sudo: false
+sudo: true
 cache: pip
 
 addons:
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
       - python-lxml # because pip installation is slow
+      - ruby-sass
```
Comment (pedrobaeza) [inline .travis.yml]: I think this is not a valid package. You have to put it in install part. But let's wait the result.

Comment (cubells) [reply .travis.yml]: @pedrobaeza  ok
@Yajo I saw it too late :(

**Comment by cubells** [comment]
@pedrobaeza Travis failed on putting ruby-sass on install section.

File: .travis.yml [hunk 2]
```diff
@@ -31,6 +30,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - ruby-sass
```
Comment (yajo) [inline .travis.yml]: apt-get install ruby-sass

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
When you migrate an addon, you have to retrieve the commit history.

In case addon does not exist in branch: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-8.0#how

And please put meaningful messages in commits.

**Pull request closed without merging**